spyder/api/widgets/mixins.py

```diff
@@ -667,6 +667,13 @@ class SpyderWidgetMixin(SpyderActionMixin, SpyderConfigurationObserver,
     # Context name used to store actions, toolbars, toolbuttons and menus
     CONTEXT_NAME = None
 
+    # Qt Web Widgets may be a heavy dependency for many packagers
+    # (e.g. conda-forge)
+    # We thus ask plugins to declare whether or not they need
+    # web widgets to enhance the distribution of Spyder to users
+    # https://github.com/spyder-ide/spyder/pull/22196#issuecomment-2189377043
+    REQUIRE_WEB_WIDGETS = False
+
```

Comment:
```suggestion
```

This constant is only required at the plugin level, so no need to have it here.

Reply:
ah thank you, i must have forgotten to delete this in my refactor.